src/lib/mod.rs

```diff
@@ -32,6 +32,7 @@
 
 #![feature(try_from)]
 
```

Comment:
Ah, no, this is intentional: it's a feature scoped over the entire crate and not connected to one of the `extern crate` lines below.